### ISSUE TITLE
Preview/embed background color?

### ISSUE THREAD

**Issue by chris-dura** [amccloud/storybook-addons-abstract#20]
Is there a way to change the embed preview from the dark background to a lighter background?

<img width="1222" alt="Screen Shot 2020-07-16 at 2 00 47 PM" src="https://user-images.githubusercontent.com/3680914/87729379-76247c80-c782-11ea-8d11-36d5b8c531b8.png">

**Comment by amccloud** [comment]
No way to control it yet, the embed is picking up your system's light or dark preference

I'll be adding more options around custom backgrounds in the next release. Can you tell me more about your use case?

**Comment by chris-dura** [comment]
@amccloud -- We have several themes for our components, but basically the components can either appear over a `light` background (usually `white` or light gray), or the components appear over `dark` backgrounds. So, in terms of the addon, there is a layer for each them for the components... iow, there `password-light` and `password-dark` layers in our Abstract Collection/Layers. So, I can obviously link to either; however, if the background changes based on the OS setting, the only way I'd be able to handle that now is to maybe create multiple stories, which isn't really ideal :/

I imagine, the dirt simple way to avoid this is to just provide a per-config way to override the OS setting...

```json
parameters: {
  abstract: {
    url: 'https://share.abstract.com/fjoweijwelbBLahblah',
    appearance: 'light'
  }
}
```

I think just providing a custom background option (even accepting a raw `rgba` or `hex`) would allow handling pretty much any use case in a dirt simple way... but, if you wanted to get fancy with it, it'd be kind of cool to allow "swapping" of Abstract Layers in the _same_ story, based on OS setting, or even the Storybook Theme config... so essentially addon users could display one layer if they're using `theme.dark` in their Storybook config, or another one if they're using `theme.my-random-custom-theme` 🤷

**Comment by amccloud** [comment]
@chris-dura thanks for the feedback. Next release of the addon will have a [`theme` and `chromeless` option](https://github.com/amccloud/storybook-addons-abstract/pull/22) 👍

**Comment by amccloud** [comment]
Released in 5.1.0 https://github.com/amccloud/storybook-addons-abstract#change-embed-background-color
